tiled/server/router.py

```diff
@@ -1390,6 +1372,62 @@ async def put_awkward_full(
     return json_or_msgpack(request, None)
 
 
+@router.patch("/metadata/{path:path}", response_model=schemas.PatchMetadataResponse)
+async def patch_metadata(
+    request: Request,
+    body: schemas.PatchMetadataRequest,
+    validation_registry=Depends(get_validation_registry),
+    settings: BaseSettings = Depends(get_settings),
+    entry=SecureEntry(scopes=["write:metadata"]),
+):
+    if not hasattr(entry, "replace_metadata"):
+        raise HTTPException(
+            status_code=HTTP_405_METHOD_NOT_ALLOWED,
+            detail="This node does not support update of metadata.",
+        )
+
+    if body.content_type in [
```

Comment:
In the spirit of, "Adhere to standards wherever possible," I would prefer to start with strictly accepting `application/json-patch+json` and `application/merge-patch+json` here. The use case for aliases seems narrow to me. The MIME type are unambiguous, Google-able, and easy enough to copy/paste from examples, or achieve through tooling like the Tiled Python client.

At least, I would wait to add aliases until/unless there is demonstrated demand for it that can be easily addressed another way. It's easy to add this later if we need to, but hard to remove once supported.